polly/plugins/maya/publish/submit_deadline.py

```diff
@@ -25,7 +25,8 @@ def process(self, instance):
         from avalon import api
         from avalon.vendor import requests
 
-        assert api.Session["AVALON_DEADLINE"], "Requires AVALON_DEADLINE"
+        deadline = api.Session.get("AVALON_DEADLINE", None)
```

Comment:
And in fact, we could skip the `.get()` and `assert` and just attempt to get the key directly. An error will be thrown with a similar message, and we'll be two lines shorter.

What do you think?